lib/mocha/expectation.rb

```diff
@@ -149,7 +149,6 @@ def at_least(minimum_number_of_times)
     #   # => verify fails
     def at_least_once
       at_least(1)
-      self
```

Comment:
As per my earlier comment, I think we might want to leave these methods returning `self`. While the calls to cardinality-related methods on `Expectation` are often the last in a chain, I suspect that isn't always the case. Also it feels inconsistent to only make this change to these two cardinality-related methods and not the others, but maybe I'm missing something.

Perhaps this highlights some missing acceptance tests...?

Reply:
Sorry. I'd completely missed that these methods call other methods on `Expectation` which themselves already return `self`. In my haste I somehow assumed they were calling methods on `Cardinality`.